frontend/package.json

```diff
@@ -31,6 +32,7 @@
     "papaparse": "^5.3.1",
     "pinia": "^2.0.12",
     "qs": "^6.11.0",
+    "rollup": "^2.50.0",
```

Comment:
Do we need `rollup`?🤔️

Reply:
I failed when running `pnpm i canvas-confetti` and got an error message about phantom depending rollup. So I have to add it to the dependencies.